### PR TITLE
feat: used api on server

### PR DESCRIPTION
**Pull request by Aleksey28**
1. added owner, repo, token in global space
2. added getting list workflows runs on the server

### CODE REVIEW

File: src/globals.js [hunk 1]
```diff
@@ -2,15 +2,25 @@ import { debugGlobal } from './config.js';
 
 export let host = 'localhost';
 export let port = 1337;
+export let owner = '';
```
Comment (AndreyBelym) [inline src/globals.js:5]: Shouldn't we cover new properties in existing tests too?

**Review by AndreyBelym — COMMENTED** [review]
That's a good one, but let's try to not decrease our test coverage just from the start. You can also look at the IstanbulJS and `nyc` to automate calculating test coverage for your project. Unfortunately, TestCafe's codebase is huge, so we should put a lot of effort to integrate IstanbulJS. So this point is highly optional.

**Comment by Aleksey28** [comment]
FPR

**Comment by Aleksey28** [comment]
FPR

**Comment by Aleksey28** [comment]
FPR

**Review by AndreyBelym — DISMISSED** [review]
👏

**Review by AndreyBelym — APPROVED** [review]

**Pull request merged**